api/openapi.gen.json

```diff
@@ -286,6 +250,30 @@
           }
         }
       }
+    },
+    "/sources/{ID}/instance_types": {
+      "get": {
+        "description": "Return a list of instance types",
+        "operationId": "getInstanceTypeList",
```

Comment:
Should I add info about the ID? @ezr-ondrej @lzap

Reply:
I wonder why it is not generated by itself, but sure, I believe we should have type and description for the ID